docker/muranoservice/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+mysql:
+   image: mysql:latest
+   container_name: mysql
+   ports:
+   - 3306:3306
+
+   environment:
+     MYSQL_DATABASE: murano
+     MYSQL_USER: mysql
+     MYSQL_PASSWORD: mysql
+     MYSQL_ROOT_PASSWORD: supersecret
+rabbit:
+   image: rabbitmq:3
+   container_name: rabbit
+   ports:
+   - 5672:5672
+murano:
+  image: murano-service
```

Comment:
I thought that the name of the image was muranoservice, at least it is written in the documentation in this way, isn't?